ingress-{{cookiecutter.repo_name}}/conf.ini

```
@@ -0,0 +1,11 @@
+[Logging]
+configuration_file = log.conf
+
+[Azure Storage]
+ingress_url = https://dp-{{cookiecutter.environment}}.westeurope.cloudapp.azure.com/osiris-ingress
+filesystem_name = datasets
+
+[Datasets]
+source = {{cookiecutter.name}}
```

Denne skal vidst rettes til:
```suggestion
source = {{cookiecutter.dataset_guid}}
```